pages/item-new/_.vue

```diff
@@ -33,46 +33,52 @@
 
           Keep/reuse client-only?
       -->
-        <client-only />
+        <client-only>
+          <ItemHero
+            :identifier="identifier"
+            :media="media"
+            :use-proxy="useProxy"
+            :data-provider-name="dataProvider.values[0]"
+            :data-provider-lang="dataProvider.code"
+            :is-shown-at="isShownAt"
+          />
+        </client-only>
       </b-container>
       <b-container>
-        <b-row class="mb-3">
-          <b-col>
-            <!-- TODO: add new title + description box
-              Useful data:
-
-              titlesInCurrentLanguage
-              descriptionInCurrentLanguage
-            -->
+        <b-row class="mb-3 justify-content-center">
+          <b-col cols="10">
```

Comment:
Could you change this (and everywhere else on this page) to 

`<b-col cols="12" class="col-lg-10">`

So it takes the full width on smaller screens.

Reply:
oh yes, good point